platforms/web/src/App.tsx

```diff
@@ -200,6 +200,14 @@ function App() {
                                 Add mention
                             </button>
                         )}
+                        {suggestion && suggestion.key === 2 && (
+                            <button
+                                type="button"
+                                onClick={(_e) => wysiwyg.command('/spoiler ')}
+                            >
+                                Add /spoiler command
+                            </button>
+                        )}
```

Comment:
This could be neatened up with a util to determine the key. Potentially worth exporting too so that it can be used in Element web. Will give that some thought over the weekend.

Reply:
I have not added an exported util, I've just slightly changed the suggestion that is exposed to React to include a character and a type to make it easier to handle in element-web